### ISSUE TITLE
Track train/validation metrics in the Sagemaker dashboard when doing remote tuning

### ISSUE THREAD

**Issue by 610v4nn1** [awslabs/syne-tune#563]
**Is your feature request related to a problem? Please describe.**
Sagemaker customers using Renate would like to see metrics being updated in the Sagemaker console page available for each training job. This requires also to aggregate metrics when running a local tuning job with multiple workers within the same training job.

**Describe the solution you'd like**
Training and validation metrics should be displayed in the Sagemaker console (see https://docs.aws.amazon.com/sagemaker/latest/dg/training-metrics.html -- in particular the section "Monitoring Training Job Metrics (SageMaker Console)") 

**Describe alternatives you've considered**
We can discuss this offline.

**Comment by mseeger** [comment]
As for metrics output during training, this is the responsibility of the training code. We provide examples for SM backend.

**Comment by mseeger** [comment]
One interesting thing we do not have, would be to emit metrics to the dashboard for remote tuning. That is a good point

**Comment by 610v4nn1** [comment]
Can you link the example you mention?

**Comment by mseeger** [comment]
https://github.com/awslabs/syne-tune/blob/main/examples/launch_height_sagemaker.py#L84

**Comment by mseeger** [comment]
It is the `metric_definitions` field for a SM estimator.
Of course, you also need to report these metrics. But we do that with the `Reporter` anyway.

**Comment by mseeger** [comment]
I put the idea to output metrics for HPO experiment for remote tuning in our backlog, because I like it. I'll probably do this soon. This will then work if you use remote tuning with local backend, for example.

**Comment by mseeger** [comment]
OK, this means you use remote tuning, and so you'll need what I wrote as a task into our backlog.
Maybe you could open an issue and give some more details of what metrics you'd like to be reported? I can then work on it, this is a good feature to have.

**Comment by 610v4nn1** [comment]
I updated the issue title instead of opening a new one.

We need metrics which are tied to the metric optimized with HPO (e.g., best so far, value in the last evaluation), and a convention for customers to define their own (e.g., all the metrics provided to the syne-tune reporter). On top of that I think it *would be nice to have* metrics which are tied to HPO optimizers internals and can be optimizer-specific (e.g., num configurations completed, num configurations pruned).

**Comment by mseeger** [comment]
PR #590 will close this issue

**Comment by mseeger** [comment]
@610v4nn1 The PR is offering most of what you asked for. We have a backlog item to configure a CloudWatch setup so these metrics are displayed nicely. Right now, the docs offer some guidance.
